packages/schema/docs/build/schema.md

```diff
@@ -39,6 +39,8 @@ This is automatically generated by the `npm run docs` command in `zapier-platfor
 * [/HydratorsSchema](#hydratorsschema)
 * [/KeySchema](#keyschema)
 * [/MiddlewaresSchema](#middlewaresschema)
+* [/ReadBulkSchema](#readbulkschema)
```

Comment:
Will this be publicly available to users? If not, we can hide it from the docs.

Reply:
Good call - no, I don't want this to be broadcast just yet. Removing